src/Microsoft.IdentityModel.JsonWebTokens/JsonWebToken.cs

```diff
@@ -151,7 +156,7 @@ public virtual IEnumerable<Claim> Claims
                 {
                     if (entry.Value == null)
                     {
-                        claims.Add(new Claim(entry.Key, string.Empty, JsonClaimValueTypes.JsonNull, issuer, issuer));
+                        claims.Add(new Claim(entry.Key, String.Empty, JsonClaimValueTypes.JsonNull, issuer, issuer));
```

Comment:
this should be lower case S 'string' not 'String'

Reply:
Changed all occurrences of String.Empty to string.Empty in this file.